### DISCUSSION TITLE
Stop words for taxonomies

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-taxonomies#1750] [Feature Requests]
Related to https://github.com/TaxoPress/TaxoPress/issues/1376

> 
authors constantly create idiotic tags, we need a blacklist

**Comment by stevejburge** [comment]
Again, could be a good Pro feature

**Comment by ojopaul** [comment]
@stevejburge How does this work?

I feel this will really be complicated especially with error message handler since user can create taxonomy terms from different screen including post and term screen.

**Comment by stevejburge** [comment]
OK, we can leave this for now. This milestone is already busy.
